README.md

```diff
@@ -28,11 +28,14 @@ The site was built for the CS98 course at Dartmouth College.
 
 ## Run Locally
 1. Start up a local server: ```python manage.py server```
-2. Start up the scraper: ```python manage.py scrape```
+2. In another terminal session,
```

Comment:
I don't think it really matters how we write out the readme because no one will be using it other than us. However, I find it's a pain personally to set up the queueing and everything locally. Especially since the tasks put into the queue don't go away when you shut down the server. When you turn the server back on, the events continue. My "ideal" way to use the app is with a nonlocal scrape once. It makes it easier.